connector/spanmetricsconnectorv2/internal/aggregator/aggregator.go

```diff
@@ -27,6 +27,11 @@ import (
 	"go.opentelemetry.io/collector/pdata/pmetric"
 )
 
+const (
+	scopeName            = "otelcol/spanmetricsconnectorv2"
```

Comment:
non-blocker: do we want a longer scope name like the hostmetrics one? Or is it better to keep "elastic repo" out of the picture?

Reply:
Yup, that will be a followup - didn't want to mix PRs.